Makefile

```diff
@@ -14,7 +14,9 @@ TAG = lorax-$(VERSION)-$(RELEASE)
 IMAGE_RELEASE = $(shell awk -F: '/FROM/ { print $$2}' Dockerfile.test)
 
 ifeq ($(TEST_OS),)
-TEST_OS = fedora-30
+OS_ID = $(shell awk -F= '/^ID/ {print $$2}' /etc/os-release)
+OS_VERSION = $(shell awk -F= '/^VERSION_ID/ {print $$2}' /etc/os-release)
+TEST_OS = $(OS_ID)-$(OS_VERSION)
```

Comment:
@bcl I kind of like this. I will also be helpful for testing internal snapshots. It just sets TEST_OS to the OS version on the host, which for me is 1 less thing to think about. In CI TEST_OS is explicitly defined anyway.